app/models/pame_evaluation.rb

```diff
@@ -83,62 +83,46 @@ def self.parse_filters(filters)
     site_ids = []
     country_ids = []
     where_params = {sites: "", methodology: "", year: "", iso3: ""}
-    filters.each do |filter|
-      options = filter["options"]
-      case filter['name']
-      when 'iso3'
-        countries = options
-        site_ids << countries.map{ |iso3| Country.find_by(iso_3: iso3).protected_areas.pluck(:id) }
-        where_params[:sites] = site_ids.flatten.empty? ? nil : "protected_area_id IN (#{site_ids.join(',')})"
-        country_ids << countries.map{ |iso3| "#{ Country.find_by(iso_3: iso3).id }" }
-        where_params[:iso3] = country_ids.flatten.empty? ? nil : "countries.id IN (#{country_ids.join(',')})"
-      when 'methodology'
-        options = options.map{ |e| "'#{e}'" }
-        where_params[:methodology] = options.empty? ? nil : "methodology IN (#{options.join(',')})"
-      when 'year'
-        where_params[:year] = options.empty? ? nil : "year IN (#{options.join(',')})"
+    if filters.nil?
+      {}
+    else
+      filters.each do |filter|
+        options = filter["options"]
+        case filter['name']
+        when 'iso3'
+          countries = options
+          site_ids << countries.map{ |iso3| Country.find_by(iso_3: iso3).protected_areas.pluck(:id) }
+          where_params[:sites] = site_ids.flatten.empty? ? nil : "protected_area_id IN (#{site_ids.join(',')})"
+          country_ids << countries.map{ |iso3| "#{ Country.find_by(iso_3: iso3).id }" }
+          where_params[:iso3] = country_ids.flatten.empty? ? nil : "countries.id IN (#{country_ids.join(',')})"
```

Comment:
Please correct me if I'm wrong, but essentially, if we have already fetched the PA ids we need to filter by, there's probably no need to check for countries ids as well because we would already have the set of PAs needed.
Also, I can't find where `where_params[:iso3]` is actually used